src/ExtendDevSite.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
```

Comment:
This is a testing file that though exported through lib, is not used when consumers use the TerraDevSite plugin.
This file duplicates TerraDevSite.jsx but imports the config from build/terra-application/siteConfig file.